charter.md

```diff
@@ -53,15 +53,164 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
```

Comment:
This is a fairly big deal. Current members are no longer allowed to run again without ever having violated the CoC or going through any judicial governance process.

In fact, 4 members of the steering committee could resign at the same time and not say why. This poison pill would stop people from running in the new election and the community would never know the reason. Did the people who didn't resign do something wrong? It wasn't CoC based. What's going on? Community members may not know.

This is not transparent (which is a k8s principle).

Reply:
That bothers me too, but what can be done about it? We can't prevent people from resigning and if enough people resign at a time we'd have to have an election to replace them.
* Only run an election for the vacant seats? (with most popular winners getting the longest-duration seats)
* Run an election for everyone, but only prevent those who resigned from running?

I don't really think either of these options is great (not a huge fan of the option in the current revision, either, though)

Reply:
I believe the revision Matt is asking for is to remove the part that says that nobody on the dissolved SC can run in the subsequent election.  I agree with that critique.  Among other things, it could leave us in a situation where we don't have enough candidates to reform the SC.

Reply:
> This is not transparent (which is a k8s principle).

FWIW we are where we are today because of folks not adhering to k8s principles - @mattfarina - hope you would be up to call out that set of behaviors as well Matt.

Reply:
(Still noodling on this section...)

Reply:
We operate in a very diverse area around a technical project.

To illustrate diversity we can look at ideology. We have people with varying belief backgrounds including atheist, christian, Muslim, Hindu, and many others. Then there are the political ones such as Renaissance, Socialist Party, Free Voters, National Renewal, Republicans, and Democrats. I started by listing non-US political parties because we are an international crowd with contributions from so many different countries. It can be easy to miss this with a North American based steering committee (only one member outside the US and that one is still in North America).

Everyone comes together for the technology (i.e. Kubernetes). Even when they oppose each other in some other space.

This is why things like transparency and inclusiveness are so important (a value and a principle), IMHO. Including people in the technical conversations and activities of the project no matter their politics, religion, gender, or other characteristic. Transparency so that people in different environments, who sometimes can't pick up on the social aspects in one geo, can follow and make well informed decisions.

There is a time to not have transparency. CoC violation details is an easy example. But, for big things like removing a steering committee member for a non-CoC issue or the dissolution of the steering committee we need transparency. So this cross region, cross belief, cross political, cross primary language community can navigate what's going on.

That's just my 2 cents.

Reply:
@dims and @mattfarina I think it would be best if in this particular place we stuck to debating how to wind things down fairly and **not** why one might want to do so. Otherwise we will overtrain on the emergency of the moment and make a policy that doesn't generalize, which is even worse for the board in the long term. The time for the discussion you are having is for later, when people are trying to wield the policy described here. (TLDR this is very close to derailing the conversation here, which has so far been very productive.)

Reply:
@lavalamp fair advice. thanks!

Reply:
I have a slight lean towards striking this line, myself. But that does not address the transparency issue at the top of this thread.  Whether anyone can re-run or not, there's nothing we can do to force them to make a statement or for that statement to be useful or even true.

We could get draconian and say something like "We ask resigning members to make a statment of intent.  If a resigning member chooses to run in the new election, they must make such a statement (100 words or less)."  But that STILL doesn't make it true or relevant or sensical.

"I resigned because I needed to buy milk, but now that I have milk I am interested in serving again."

If we do keep this rule, we should maybe be extra-clear whether this applies to ALL members or just resigning members, and that those people may (?) run in future elections.

Reply:
@thockin it's not just about the resigning members, but also members who will NOT resign as well. The intent was one subsequent election. Think of when steering is fragmented enough to be dysfunctional and the folks resigning will save trouble for future members that fill their seats.